### PR TITLE
Fixed Django 1.11 related issue in test

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
None

#### What's this PR do?
Fixes CI issues where one integration test was failing due to django upgrade on edX

#### How should this be manually tested?
Travis should work

@pdpinch

### CODE REVIEW

**Comment by amir-qayyum-khan** [comment]
@pdpinch I am getting this error on all PRs. To fix error i made this PR

```
        solution = block.student_state()['solution']
>       assert '<a href="{}/test.pdf">A PDF</a>'.format(
            static_asset_path if has_static_asset_path else '/c4x/foo/bar/asset'
        ) == solution
E       AssertionError: assert '<a href="/a/...df">A PDF</a>' == '<a href="/sta...df">A PDF</a>'
E         - <a href="/a/different/static/asset/path/test.pdf">A PDF</a>
E         + <a href="/static/a/different/static/asset/path/test.pdf">A PDF</a>
E         ?          +++++++
edx_sga/tests/integration_tests.py:793: AssertionError
```

File: edx_sga/tests/integration_tests.py [hunk 1]
```diff
@@ -779,8 +780,10 @@ def test_replace_url(self, has_static_asset_path):
         If the static asset path is set on a course, it should be substituted when the course is rendered
         """
         # make a runtime with a static asset path, which will override the base_asset_url
-
-        static_asset_path = '/a/different/static/asset/path'
+        if django.VERSION >= (1, 11):
+            static_asset_path = '/static/a/different/static/asset/path'
+        else:
+            static_asset_path = '/a/different/static/asset/path'
```
Comment (pdpinch) [inline edx_sga/tests/integration_tests.py]: My understanding is that this `static_asset_path` setting is testing a particular case, and the existing value is arbitrary. 

How did django 1.11 change that the old, arbitrary path no longer work? 

Can we just change it for all versions of django, to be `static_asset_path = '/static/a/different/static/asset/path'` ?

**Comment by pdpinch** [comment]
I think we need to look at edx-platform master to see how this 'static_path' feature is tested in lms.

File: edx_sga/tests/integration_tests.py [hunk 2]
```diff
@@ -779,8 +780,11 @@ def test_replace_url(self, has_static_asset_path):
         If the static asset path is set on a course, it should be substituted when the course is rendered
         """
         # make a runtime with a static asset path, which will override the base_asset_url
-
         static_asset_path = '/a/different/static/asset/path'
+        solution_expected = '/a/different/static/asset/path'
+        if django.VERSION >= (1, 11):
```
Comment (noisecapella) [inline edx_sga/tests/integration_tests.py]: I'm not sure this conditional is the right idea here. What caused the change of behavior in Django? Maybe we should be using an absolute path for the static root in the test settings instead

**Comment by amir-qayyum-khan** [comment]
@noisecapella can you take a look. This test is casing travis to fail.

**Comment by amir-qayyum-khan** [comment]
@noisecapella i followed this example https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/tests/test_courses.py#L267 for this test

**Pull request merged**